test/chrome/chromeDebugAdapter.test.ts

```diff
@@ -427,6 +427,30 @@ suite('ChromeDebugAdapter', () => {
             });
         });
 
+        // This is needed for Edge debug adapter, please keep the signature and logic of sendLoadedSourceEvent() method intact.
+        test('tests that sendLoadedSourceEvent can accept an additional paramter to override the `reason` parameter', done => {
+            const loadedSourceEventReason: LoadedSourceEventReason = 'changed';
+            sendEventHandler = (event) => {
+                assert.equal('loadedSource', event.event);
+                assert.notEqual(null, event.body);
```

Comment:
Would the test catch more potential issues if we tested the contents of all the members of the event argument?

Reply:
Should we keep the tests atomic and follow the one-thing-at-a-time principle? Or otherwise, when the product code changes, we might have to update all the tests, which is tedious and does not gain us anything.

Reply:
Ideally the test should only fail when the contract/interface changes. If we change the implementation while maintaining the original contract/interface then the test shouldn't fail nor need updating.

Reply:
Resolved offline. @digeff  and I both agree on this change now.